ConfirmationInfoTest.java

```diff
@@ -0,0 +1,34 @@
+package com.sweng894.GetVaccinated;
+//SWEN-39 testing for use case SWEN-24
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ConfirmationInfoTest {
```

Comment:
The test cases for this seem to be empty and do not run any of the application code. I understand that one reason for this may be that the test case would be dependent upon SWEN-24, which does not have code in the main branch yet.

If that is the case, then I recommend including the unit tests for SWEN-24 in the same PR that includes the application code for SWEN-24. In the future we shouldn't need separate stories for unit testing. Unit testing is part of having the story be "done".